pkg/models/nodes.go

```diff
@@ -216,3 +217,9 @@ func getNodeFileSystemStatus(node *v1.Node) (string, string, string) {
 	}
 	return "", "", ""
 }
+
+func DrainNode(nodename string) {
```

Comment:
As no logic is added in this PR for nodes model, I think you should undo this change

Reply:
ok，i will remove this funcion